gcloud-java-storage/src/test/java/com/google/gcloud/storage/ITStorageTest.java

```diff
@@ -86,7 +86,8 @@ public static void afterClass() throws ExecutionException, InterruptedException
 
   @Test(timeout = 5000)
   public void testListBuckets() throws InterruptedException {
-    Iterator<BucketInfo> bucketIterator = storage.list(Storage.BucketListOption.prefix(BUCKET),
+    Iterator<Bucket> bucketIterator =
```

Comment:
In `ITStorageTest`, we delete every blob we create at the end of each function.  Is there any reason against getting a list of blobs in the bucket and deleting them in a tearDown function after each test?